admin/class-arlo-for-wordpress-admin.php

```diff
@@ -195,19 +195,18 @@ public function enqueue_admin_scripts() {
 		if ( ! isset( $this->plugin_screen_hook_suffix ) ) {
 			return;
 		}
-		
+
 		wp_enqueue_script( $this->plugin_slug . '-admin-global-script', plugins_url( 'assets/js/admin_public.js?20170424', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
 
 		$screen = get_current_screen();
 		if ( $this->plugin_screen_hook_suffix == $screen->id ) {
 			wp_enqueue_script( $this->plugin_slug . '-lsapiclient', plugins_url( 'assets/js/lib/ls-apiclient-1.2.0.min.js', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
-			wp_enqueue_script( $this->plugin_slug . '-jquery-ui', plugins_url( 'assets/js/lib/jquery-ui.min.js', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
```

Comment:
Can you also remove the jquery-ui.min.js and jquery-ui.min.css files if they are no longer needed? 

As well as the reference to jquery-ui.min.css at [#L176](https://github.com/ArloSoftware/arlowp-plugin/blob/bbb3ded7ec43e5d2749f33a8c6a54040128f13ba/admin/class-arlo-for-wordpress-admin.php#L176)